lib/hammer_cli_foreman_tasks/recurring_logic.rb

```diff
@@ -27,6 +27,20 @@ class CancelCommand < HammerCLIForeman::DeleteCommand
       build_options
     end
 
+    class ClearCancelledCommand < HammerCLIForeman::DeleteCommand
+      def request_params
+        params = super
+        params["search"] = "state=cancelled"
+        params
+      end
+      action :bulk_destroy
+      command_name 'clear_cancelled'
```

Comment:
Hammer commands usually use '-' instead of '_'.

Reply:
fixed thanks!